src/platform/packages/shared/kbn-unified-metrics-grid/src/components/metric_flyout/metric_flyout_content.tsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import React, { useState, useMemo } from 'react';
+import { EuiTabs, EuiTab, EuiSpacer, EuiCodeBlock } from '@elastic/eui';
+import { OverviewTab } from './overview_tab';
+import type { MetricField } from '../../types';
+import { TabTitleAndDescription } from './tab_title_and_description';
+
+const tabs = (metric: MetricField, esqlQuery: string) => [
```

Comment:
Suggestion: maybe this object should only contain the metadata, and not the components. Also, if we follow this approach, maybe it should be named `getTabs`?